pkgs/by-name/ap/applesauce/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
a bit more convenient than `rec` (you can see exactly which comes from where + you can reuse implicit attrs, such as `finalAttrs.finalPackage`)